packages/components/WhyItWorksSection.tsx

```diff
@@ -13,15 +13,15 @@ import {
 
 import { PersonalizedLearningIcon, PracticalLearningIcon, TeamworkIcon } from "@packages/components/icons";
 import { useI18n } from "@packages/features/i18n-context";
-
 import Section from "./Section";
+import { classes } from "@packages/utils/classes";
```

Comment:
I think this can be removed

Reply:
change here [8b1abae](https://github.com/podcodar/webapp/pull/182/commits/8b1abae0f26c86f07f32bf804aff05c41c698545)